### ISSUE TITLE
When using the 2k Engine version with RPG2k3Commands=1 active, SaveData will not include 2k3 data (Creates issue with spritesheets)

### ISSUE THREAD

**Issue by florianessl** [EasyRPG/liblcf#476]
This created an issue with spritesheets:
When displaying a picture using the parameters "spritesheet_cols", "spritesheet_rows", "spritesheet_frames" with set values, then saving & loading a savefile, these parameters will be set to their default values (1, 1, 0) which will result in the entire spritesheet picture being shown after a load instread of just the frame.
Examining the lsd shows that this happens while writing the savefile. Everything that has the flag "is2k3" set in fields.csv is ignored by the writer, even though it is necessary for the "RPG2k3Commands" mode.

I'll be playing around a bit with a custom build that forces 2k3 mode for liblcf to test if this has any side effects. But maybe a third intermediate mode for the SaveFile writer would be necessary for this case.

**Comment by Ghabry** [comment]
Yeah, the INI is not parsed. Maybe should be considered. You can use argument ``--2k3`` to force 2k3 mode.
